### ISSUE TITLE
Change the maplabel's custom CSS after click

### ISSUE THREAD

**Issue by skinkie** [sothawo/mapjfx#28]
I'm trying to prototype some dynamic behavior with maplabels. How can I dynamically update the CssClass after click? At this moment it remains the same (initial) CssClass.

```java
mapView.addEventHandler(MapLabelEvent.MAPLABEL_CLICKED, event -> {
            event.consume();
            MapLabel mapLabel1 = event.getMapLabel();
            mapLabel1.setCssClass("green-label");
        });
```

This seems to work, but is rather wastefull.
```java
        mapView.addEventHandler(MapLabelEvent.MAPLABEL_CLICKED, event -> {
            event.consume();
            MapLabel mapLabel1 = event.getMapLabel();
            MapLabel mapLabel2 = new MapLabel(mapLabel1.getText()).setPosition(mapLabel1.getPosition()).setVisible(true).setCssClass("green-label");
            mapView.removeLabel(mapLabel1);
            mapView.addLabel(mapLabel2);
        });
```

**Comment by sothawo** [comment]
started work in branch feature/change-label-css

**Comment by sothawo** [comment]
implemented in develop branch. To see it in action, check out develop and run the test program with `mvn test exec:java -Dexec.mainClass="com.sothawo.mapjfx.TestApp" -Dexec.classpathScope=test`. Click on _add Marker_ and the move your mouse over the Label.

The sample uses the ENTERED and EXITED events you implemented to change the label's css.

**Comment by sothawo** [comment]
implemented in release 1.14.0
